src/openrct2-ui/windows/TitleScenarioSelect.cpp

```diff
@@ -442,7 +442,7 @@ static void window_scenarioselect_paint(rct_window* w, rct_drawpixelinfo* dpi)
                                                                                      : STR_WINDOW_COLOUR_2_STRINGID;
 
     // Text for each tab
-    for (int32_t i = 0; i < 8; i++)
+    for (int32_t i = 0; i < SCENARIO_CATEGORY_COUNT; i++)
```

Comment:
`std::size(ScenarioOriginStringIds)` is probably better, since it more explicitly makes sure not to loop beyond the size of `ScenarioOriginStringIds`.

Reply:
Done